src/main/java/com/textmessenger/service/PostServiceImpl.java

```diff
@@ -80,6 +81,15 @@ public void deletePost(Post post) {
     return PostToFront.convertListPostsToResponse(postRepository.findAll(orderBy()));
   }
 
+  @Override
+  public List<PostToFront> getPage(int page, int size) {
+    List<Post> all = postRepository.findAll(orderBy());
+    return PostToFront.convertListPostsToResponse(all.stream()
+            .skip(page * size)
```

Comment:
![Bug](https://sonarsource.github.io/sonar-github/bug.png 'Bug') Bug: Cast one of the operands of this multiplication operation to a "long". ([squid:S2184](https://sonarcloud.io/organizations/final_project_text_messenger/rules#rule_key=squid%3AS2184))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=Text_Messenger_Project&pullRequest=169&issues=AWWmV_xpfVFgtsXaw25h&open=AWWmV_xpfVFgtsXaw25h)

[ISSUE_KEY]: AWWmV_xpfVFgtsXaw25h